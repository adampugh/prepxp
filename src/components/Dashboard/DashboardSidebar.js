import React from "react";
import { Link } from "react-router-dom";

import desk from "../../assets/images/deskimg.png";

const dashboardSidebar = () => (
    <div className="sidebar">
        <img src={desk} alt="man sitting at a desk" />
        <div className="sidebar__panel">
            
            <p>Create a new list of interview questions</p>
            <Link to="/list/edit">
                <button className="btn">Create</button>
            </Link>
        </div>
        <div className="sidebar__panel">
            <img src={desk} alt="man sitting at a desk" />
            <p>Add a list of interview questions for our seleted lists</p>
            <Link to="/search">
                <button className="btn">Lists</button>
            </Link>
        </div>
        <div className="sidebar__panel">
            <img src={desk} alt="man sitting at a desk" />
            <p>Read the latest tips and get the job you deserve</p>
            <Link to="/blog">
                <button className="btn">Blog</button>
            </Link>
        </div>
    </div>
);

export default dashboardSidebar;