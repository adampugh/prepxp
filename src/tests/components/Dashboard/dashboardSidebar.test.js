import React from "react";
import { shallow } from "enzyme";

import DashboardSidebar from "../../../components/Dashboard/DashboardSidebar";

describe("<DashboardSideBar />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<DashboardSidebar />);
        expect(wrapper).toMatchSnapshot();
    });
});