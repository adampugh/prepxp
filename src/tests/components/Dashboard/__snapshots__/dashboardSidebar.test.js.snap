// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardSideBar /> should match snapshot 1`] = `
<div
  className="sidebar"
>
  <img
    alt="man sitting at a desk"
    src="deskimg.png"
  />
  <div
    className="sidebar__panel"
  >
    <p>
      Create a new list of interview questions
    </p>
    <Link
      replace={false}
      to="/list/edit"
    >
      <button
        className="btn"
      >
        Create
      </button>
    </Link>
  </div>
  <div
    className="sidebar__panel"
  >
    <img
      alt="man sitting at a desk"
      src="deskimg.png"
    />
    <p>
      Add a list of interview questions for our seleted lists
    </p>
    <Link
      replace={false}
      to="/search"
    >
      <button
        className="btn"
      >
        Lists
      </button>
    </Link>
  </div>
  <div
    className="sidebar__panel"
  >
    <img
      alt="man sitting at a desk"
      src="deskimg.png"
    />
    <p>
      Read the latest tips and get the job you deserve
    </p>
    <Link
      replace={false}
      to="/blog"
    >
      <button
        className="btn"
      >
        Blog
      </button>
    </Link>
  </div>
</div>
`;
